lib/services/app.js

```diff
@@ -0,0 +1,12 @@
+import express from 'express';
+
+import errorHandler, { notFound } from '../middleware/error_handler';
```

Comment:
I'm not sure that it is correct import

Reply:
[Resolved](https://github.com/inst-event-emitter/eventemitter-backend/pull/1/commits/29ee8781536f4769deee22e885fe69b0ccf58ca9#diff-4c20e13ad2ba66d12b169e0a032a9504L3)